### PR TITLE
DL: Add new columns in fit output summary table.

### PR DESCRIPTION
**Pull request by njayaram2**
JIRA: MADLIB-1319

Columns `dependent_vartype` and `normalizing_const` are useful to have
in the model summary table since they can be used in predict.
1) For instance, `dependent_vartype` will be useful in
https://issues.apache.org/jira/browse/MADLIB-1315 while creating the
prediction output column. In case of response, the output column should
be of the same type as the dependent variable in train data.
2) While predicting, `normalizing_const` will be useful to normalize the
test data using the same normalizing constant that was used to normalize
the train data.

Co-authored-by: Nandish Jayaram <njayaram@apache.org>

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/811/

File: src/ports/postgres/modules/deep_learning/madlib_keras_predict.py_in [hunk 1]
```diff
@@ -60,7 +61,8 @@ def predict(schema_madlib, model_table, test_table, id_col, model_arch_table,
     input_shape = get_input_shape(model_arch)
     compile_params = "$madlib$" + compile_params + "$madlib$"
     model_summary_table = add_postfix(model_table, "_summary")
-    class_values, _ = get_class_values_and_type(model_summary_table)
+    class_values, _ = get_col_value_and_type(
```
Comment (khannaekta) [inline src/ports/postgres/modules/deep_learning/madlib_keras_predict.py_in]: Instead of calling `get_col_value_and_type()` here, can we do something similar to what we have in [madlib_keras.py_in:202](https://github.com/apache/madlib/pull/365/files#diff-4d3eba8c7b20343818ac24e5730779e7R202)

File: src/ports/postgres/modules/utilities/validate_args.py_in [hunk 2]
```diff
@@ -365,6 +365,24 @@ def get_cols_and_types(tbl):
     return list(zip(col_names, col_types))
 # -------------------------------------------------------------------------
 
+def get_col_value_and_type(table_name, column_name):
```
Comment (khannaekta) [inline src/ports/postgres/modules/utilities/validate_args.py_in]: We could extend this function to handle multiple column_names in addition to a single column_name.

Comment (njayaram2) [reply src/ports/postgres/modules/utilities/validate_args.py_in]: That's a good suggestion, but it may not be necessary at the moment. If we see this function getting used more often in the future, let's refactor it out and update it this way.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/madlib-pr-build/812/
